### ISSUE TITLE
Maybe an idea about those wallets who get around 85% blocks on every shards

### ISSUE THREAD

**Issue by sebastienturr** [seeleteam/go-seele#1077]
Hi dev team.
Firstable you are doing a great job... i am here since block number 75 000 ... i know what you did ... Where we were... It is very good since the beginning and you always find solutions.

I have a strange feeling about those wallets who get a lot of incomes, seems really strange even with a massive amount of big i9 servers ... 

And i had an idea about that.
Do you think it is possible to a guy, a dev or a team dev to change the way that p2p works in the node, then only make p2p to a small amount of IPs, Like if a guy can only make a p2p with his machines on his network. That way he can keep his machines sync to start the next block and slow the other. If it's like 85% of mined blocks,  maybe some of his machines are doing normal p2p to not destory the project. 

What do you think about that ?

**Comment by shawnxin89** [comment]
> Hi dev team.
> Firstable you are doing a great job... i am here since block number 75 000 ... i know what you did ... Where we were... It is very good since the beginning and you always find solutions.
> 
> I have a strange feeling about those wallets who get a lot of incomes, seems really strange even with a massive amount of big i9 servers ...
> 
> And i had an idea about that.
> Do you think it is possible to a guy, a dev or a team dev to change the way that p2p works in the node, then only make p2p to a small amount of IPs, Like if a guy can only make a p2p with his machines on his network. That way he can keep his machines sync to start the next block and slow the other. If it's like 85% of mined blocks, maybe some of his machines are doing normal p2p to not destory the project.
> 
> What do you think about that ?

Hi, first of all thank you for your continuous support for the project. Miners may do selfish mining as you said. But they can only do that when they have a strong mining power. P2P is important for mining. We are continuously improving it so that all miners are competing fairly.

**Comment by shawnxin89** [comment]
> > Hi dev team.
> > Firstable you are doing a great job... i am here since block number 75 000 ... i know what you did ... Where we were... It is very good since the beginning and you always find solutions.
> > I have a strange feeling about those wallets who get a lot of incomes, seems really strange even with a massive amount of big i9 servers ...
> > And i had an idea about that.
> > Do you think it is possible to a guy, a dev or a team dev to change the way that p2p works in the node, then only make p2p to a small amount of IPs, Like if a guy can only make a p2p with his machines on his network. That way he can keep his machines sync to start the next block and slow the other. If it's like 85% of mined blocks, maybe some of his machines are doing normal p2p to not destory the project.
> > What do you think about that ?
> 
> Hi, first of all thank you for your continuous support for the project. Miners may do selfish mining as you said. But they can only do that when they have a strong mining power. P2P is important for mining. We are continuously improving it so that all miners are competing fairly.

To be clear, miners may do selfish mining on any POW chain. It is just like doing mining locally. To get more rewards, mining power is still the most important factor.

**Comment by frode88** [comment]
> > > Hi dev team.
> > > Firstable you are doing a great job... i am here since block number 75 000 ... i know what you did ... Where we were... It is very good since the beginning and you always find solutions.
> > > I have a strange feeling about those wallets who get a lot of incomes, seems really strange even with a massive amount of big i9 servers ...
> > > And i had an idea about that.
> > > Do you think it is possible to a guy, a dev or a team dev to change the way that p2p works in the node, then only make p2p to a small amount of IPs, Like if a guy can only make a p2p with his machines on his network. That way he can keep his machines sync to start the next block and slow the other. If it's like 85% of mined blocks, maybe some of his machines are doing normal p2p to not destory the project.
> > > What do you think about that ?
> > 
> > 
> > Hi, first of all thank you for your continuous support for the project. Miners may do selfish mining as you said. But they can only do that when they have a strong mining power. P2P is important for mining. We are continuously improving it so that all miners are competing fairly.
> 
> To be clear, miners may do selfish mining on any POW chain. It is just like doing mining locally. To get more rewards, mining power is still the most important factor.

Hi. TBH i saw many times in my wallet. That i get reward 6 then balance after few second the reward disappears.

**Comment by Vibrio787** [comment]
> To be clear, miners may do selfish mining on any POW chain. It is just like doing mining locally. To get more rewards, mining power is still the most important factor.

Can you please clarify if multiple machines can be consolidated as a pool to run as one node?  
Also what are the main determining factor/s in generating higher hash rate / mining power?

**Comment by frode88** [comment]
@shawnxin89 
just uploaded video to telegram channel. you can watch:D

**Comment by sebastienturr** [comment]
> > Hi dev team.
> > Firstable you are doing a great job... i am here since block number 75 000 ... i know what you did ... Where we were... It is very good since the beginning and you always find solutions.
> > I have a strange feeling about those wallets who get a lot of incomes, seems really strange even with a massive amount of big i9 servers ...
> > And i had an idea about that.
> > Do you think it is possible to a guy, a dev or a team dev to change the way that p2p works in the node, then only make p2p to a small amount of IPs, Like if a guy can only make a p2p with his machines on his network. That way he can keep his machines sync to start the next block and slow the other. If it's like 85% of mined blocks, maybe some of his machines are doing normal p2p to not destory the project.
> > What do you think about that ?
> 
> Hi, first of all thank you for your continuous support for the project. Miners may do selfish mining as you said. But they can only do that when they have a strong mining power. P2P is important for mining. We are continuously improving it so that all miners are competing fairly.

What do you think about a node who is check all IPs you got to p2p, like if you have 2000 nodes n the seele network, you have to be connected with at least 50% to start mining. I am a dev, i can understand technical answer ;) I know that 30% of IPs could be a lot but... we are sitting on a big project, big project mean big solution !

**Comment by shawnxin89** [comment]
> > > Hi dev team.
> > > Firstable you are doing a great job... i am here since block number 75 000 ... i know what you did ... Where we were... It is very good since the beginning and you always find solutions.
> > > I have a strange feeling about those wallets who get a lot of incomes, seems really strange even with a massive amount of big i9 servers ...
> > > And i had an idea about that.
> > > Do you think it is possible to a guy, a dev or a team dev to change the way that p2p works in the node, then only make p2p to a small amount of IPs, Like if a guy can only make a p2p with his machines on his network. That way he can keep his machines sync to start the next block and slow the other. If it's like 85% of mined blocks, maybe some of his machines are doing normal p2p to not destory the project.
> > > What do you think about that ?
> > 
> > 
> > Hi, first of all thank you for your continuous support for the project. Miners may do selfish mining as you said. But they can only do that when they have a strong mining power. P2P is important for mining. We are continuously improving it so that all miners are competing fairly.
> 
> What do you think about a node who is checking all IPs you got to p2p, like if you have 2000 nodes n the seele network, you have to be connected with at least 50% to start mining. I am a dev, i can understand technical answer ;) I know that 50% of IPs could be a lot but... we are sitting on a big project, big project mean big solution !

I don't think it works because other miners can't verify this mining condition when they receive the blocks.
